app/src/main/java/com/keylesspalace/tusky/components/login/LoginActivity.kt

```diff
@@ -99,11 +96,6 @@ class LoginActivity : BaseActivity() {
                 .into(binding.loginLogo)
         }
 
-        preferences = getSharedPreferences(
-            getString(R.string.preferences_file_key),
-            Context.MODE_PRIVATE
-        )
```

Comment:
Not sure why the SharedPreferences were created differently than everywhere else here?